metrics/mobile-app-metrics.md

```diff
@@ -100,12 +100,16 @@ The Storage service will store metrics data forwarded from the Mobile Metrics Se
 
 #### MongoDB
 
-<!-- TODO -->
-
 ##### Pros
 
+- Supports time series data and non-time series data
+
 ##### Cons
 
+- Support for time series data is not native thus needs modeling work
+- Grafana/Kibana cannot talk to MongoDB without ugly workarounds
```

Comment:
Good points. Definitely using Mongo with Grafana/Prometheus is no go zone.  
I got impression that when using any relational database we could develop simple panel as part of the service to showcase charts etc.

Reply:
@wtrocki +1, it is a good alternative.
But, if we go that way and have some application with business logic, I would prefer implementing an endpoint that Grafana can talk to. Let's first evaluate the current idea and then we can discuss alternatives like that.